src/ray/gcs/gcs_server/gcs_node_manager.cc

```diff
@@ -230,6 +230,12 @@ void GcsNodeManager::HandleUpdateResources(const rpc::UpdateResourcesRequest &re
     for (auto &entry : *to_be_updated_resources) {
       iter->second[entry.first] = entry.second;
     }
+    rpc::ResourceMap rpc_resources_map;
```

Comment:
just change `cluster_resources_` to use `rpc::ResourceMap`, so we don't need to covert it there?

Reply:
> just change `cluster_resources_` to use `rpc::ResourceMap`, so we don't need to covert it there?

Nice catch. All gcs::ResourceMap fields have been converted to rpc::ResourceMap.